lib/modules/datasource/nuget/v3.ts

```diff
@@ -233,4 +258,52 @@ export class NugetV3Api {
 
     return dep;
   }
+
+  @cache({
+    namespace: NugetV3Api.cacheNamespace,
+    key: (
+      _http: Http,
+      registryUrl: string,
+      packageName: string,
+      _packageVersion: string | null,
```

Comment:
Doesn't this mean that the cached data will be invalidated each time there's a new release?

Reply:
no, it's not used, it's just a required function parameter